src/server/dfly_main.cc

```diff
@@ -3,6 +3,7 @@
 // See LICENSE for licensing terms.
 //
 
+#include "absl/container/inlined_vector.h"
```

Comment:
a side comment: this function is called once a day. please take into consideration the context before optimizing.  I would not merge this PR

Reply:
It's true that this is not something to optimize for :)
But what's the cost? I don't think that the code is less readable, if anything it's more because the added `3` tells us the expected size
Don't you agree?

Reply:
I said it about myself. you guys can merge it but I think the thinking should be - there should be at least some positive reasons that required changing the code. I would not comment on this if it was the new code but changing the existing code - must have some reason (style, performance, readability).... Anyway, that's  how I choose my battles :)

Reply:
You're 100% right. We discussed this unimportant optimization yesterday but forgot to insert it. It's not important at all. I think we're all aligned, just different pedantic levels :)

Reply:
@romange Yes I agree as well. This was an offline discussion small nit which I forgot to add and the OCD kicked in.